hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -1096,6 +1096,57 @@ def hpat_pandas_series_ge_impl(self, other):
     raise TypingError('{} The object must be a pandas.series and argument must be a number. Given: {} and other: {}'.format(_func_name, self, other))
 
 
+@overload_method(SeriesType, 'idxmin')
+def hpat_pandas_series_idxmin(self, axis=None, skipna=True):
+    """
+    Pandas Series method :meth:`pandas.Series.idxmin` implementation.
+
+    .. only:: developer
+
+        Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_idxmin1
+        Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_idxmin_str
+        Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_idxmin_int
+        Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_idxmin_no
+
+    Parameters
+    -----------
+    axis :  :obj:`int`, default: None
+                *unsupported*
+    skipna:  :obj:`bool`, default: True
+                Exclude nan values when computing the result
+
+    Returns
+    -------
+    :obj:`pandas.Series` or :obj:`int` or :obj:`float`
+         returns :obj:`pandas.Series` object or :obj:`int` or :obj:`float`
+    """
+
+    _func_name = 'Method idxmin().'
+
+    if not isinstance(self, SeriesType):
+        raise TypingError('{} The object must be a pandas.series. Given self: {}'.format(_func_name, self))
+
+    if not isinstance(self.dtype, types.Number):
```

Comment:
Do we really focused on numeric data in series? No strings array supported?
Also, I would vote to avoid `self.dtype` using because I see it ambiguous. I think it is better to use `self._data.dtype` instead.

Reply:
I would like to correct a little bit: `self._data.dtype` -> `self.data.dtype`. `self` has not attribute `_data` on typing level.

Reply:
Fixed